### DISCUSSION TITLE
友情链接 | ZkeqのCoding日志

### DISCUSSION THREAD

**Discussion by zkeq** [zkeq/Coding#599] [Announcements]
https://coding.maylove.pub/link/

友情链接

**Comment by zkeq** [comment]
## 交换友链的话

#### 可以在此回帖

###### 发帖格式:

> 昵称:  `(你的昵称)`
>
> 头像: `(你的头像, 为了防止失效,添加时会转移到本网站内容库中)`
>
> 地址: `(请填写https,谢谢!)`
>
> 简介: `(可以一句话介绍你自己)`
>

**Reply by flow2000** [reply]
昵称: 心流

头像: https://blog.panghai.top/img/avatar.png

地址: https://blog.panghai.top/

简介: 保持谨慎，保持乐观

**Reply by zkeq** [reply]
抱歉，最近开学很忙 外加在赶项目 刚看到消息....已添加

**Comment by DERMTOP** [comment]
昵称:独木君（Derm）
网站名称:纳兰戏梦
头像: https://z3.ax1x.com/2021/03/08/6lz4Dx.jpg
地址: https://derm.top/
简介: 从未落时开始，于伊洛时结束。

**Comment by zkeq** [comment]
> 昵称:独木君（Derm） 网站名称:纳兰戏梦 头像: https://z3.ax1x.com/2021/03/08/6lz4Dx.jpg 地址: https://derm.top/ 简介: 从未落时开始，于伊洛时结束。

wow，大佬

**Comment by dongyubin** [comment]
昵称：斌仔
网站名称：文武科技柜
网站地址：https://www.wangdu.site/
描述：文武科技社的后花园
头像：https://www.wangdu.site/wp-content/uploads/2021/08/favicon.png

**Reply by zkeq** [reply]
wow 抱歉，近期在做项目 没看这个页面，这就加上

**Reply by zkeq** [reply]
已经加上咯

**Reply by zkeq** [reply]
为什么在贵站并没有看到本站的链接..

**Reply by dongyubin** [reply]
能否发一些你的友联格式，我这边好加一下

**Reply by zkeq** [reply]
大学开学了（
最近在军训....
没怎么看博客 格式什么的可以随便加...
头像的话可以用我博客的直链 CDN 无防盗链
https://img.onmicrosoft.cn/Avatar.jpg

**Comment by lyboy-wp** [comment]
站点名称：今夕何夕兮
站点描述：今夕何夕，尽三千繁华，见此良人，皆虚梦一场
站点地址：https://www.lyboy.top/
站点图标：https://lyboy.oss-accelerate.aliyuncs.com/wordpress/logo1.png

**Reply by zkeq** [reply]
抱歉，最近开学很忙 外加在赶项目 刚看到消息....已添加
贵站访问出现 cf 提示 1020 错误代码，请尽快修复

**Comment by i-abc** [comment]
大佬我加您了，对您的自建不蒜子印象尤为深刻。

昵称: 是非题
头像: https://other.shifeiti.com/logo/logo.png
地址: https://www.shifeiti.com
简介: 记录生活，热爱跑步，学习 FPGA 中

**Reply by zkeq** [reply]
抱歉，最近开学很忙 外加在赶项目 刚看到消息....已添加

**Reply by zkeq** [reply]
大佬博客好干净好喜欢

**Reply by i-abc** [reply]
> 抱歉，最近开学很忙 外加在赶项目 刚看到消息....已添加

小事小事，忙自己的项目更重要；还请多多关照

**Reply by i-abc** [reply]
> 大佬博客好干净好喜欢

这是主题作者的功劳，靠他开发我才能用上这个主题，你也可以看看 [hexo-theme-stellar](https://github.com/xaoxuu/hexo-theme-stellar)

**Reply by zkeq** [reply]
好棒，试试看

**Comment by lyboy-wp** [comment]
已添加
















在 2022-09-30 09:01:02，"i-abc" ***@***.***> 写道：

大佬我加您了，对您的自建不蒜子印象尤为深刻。

昵称: 是非题
头像: https://other.shifeiti.com/logo/logo.png
地址: https://www.shifeiti.com
简介: 记录生活，热爱跑步，学习 FPGA 中

**Comment by weekdaycare** [comment]
昵称: 涛声依旧
头像: https://pic1.imgdb.cn/item/630c05de16f2c2beb1d86de2.jpg
地址: https://weekdaycare.cn/
（域名正在备案中，暂时还没法访问）
简介: 永远相信美好的事情即将发生

**Reply by zkeq** [reply]
抱歉，最近开学很忙 外加在赶项目 刚看到消息....已添加

**Reply by zkeq** [reply]
大佬博客是好看的 👀

**Reply by weekdaycare** [reply]
٩(๑′0`๑)۶

**Comment by lbnliboning** [comment]
昵称：bill
头像：https://s2.232232.xyz/static/117/2022/12/27-63aad8a6af8d4.png
地址：http://1703.site
简介：分享网络干货和我的生活~

**Reply by zkeq** [reply]
已加，贵站的样式 仍需美化！

**Reply by zkeq** [reply]
🤔 没在网站看到本站的链接 本条进入审核期~

**Reply by ovoutr** [reply]
blog.1793.site

**Comment by wangkai000** [comment]
方方群里的群友，友链申请互换，你的我已添加～
博客名称：天渺studio
博客描述：天渺工作室的小站
博客主页地址：https://tianmiao.site
博客logo：https://s21.ax1x.com/2024/12/22/pAXtJat.jpg

**Comment by Elykia093** [comment]
我的名称: Elykia
网站地址: https://blog.elykia.cn/
描述: 致以无暇之人
头像: https://bu.dusays.com/2024/10/25/671b2438203a6.gif
站点截图: https://bu.dusays.com/2024/10/25/671b50dc2dd2b.png

**Reply by zkeq** [reply]
收到收到, 正在添加

**Comment by Ginc-c** [comment]
网站今夕何夕兮   站点图标已更换为：https://cdn.jsdelivr.net/gh/Ginc-c/Ginc/Ginc.png

**Comment by memnotop** [comment]
Name:memnotop
Avatar:https://memno.top/_astro/avatar.C1wqNjWy.webp
Link:https://memno.top/
Desc:如切如磋，如琢如磨。
